src/BulletChartSettingsModel.ts

```diff
@@ -7,6 +7,9 @@ import IEnumMember = powerbi.IEnumMember;
 import {BulletChartOrientation} from "./BulletChartOrientation";
 import ILocalizationManager = powerbi.extensibility.ILocalizationManager;
 import {BarRectType} from "./dataInterfaces";
+import FormattingId = powerbi.visuals.FormattingId;
+
+const nameof = <T>(name: Extract<keyof T, string>): string => name;
```

Comment:
what is the purpose of this?

Reply:
`nameof` function returns the name of the property in object. So if developer renames the property, Typescript will return renamed property.